frontend/src/components/Result.js

```diff
@@ -0,0 +1,131 @@
+      if (err) {
+        console.log(err);
+        return;
+      }
+      console.log(response);
+      const posts = response.getInstaPostsList();
+      const postdata = posts
+        .map(posts => ({
+          id: posts.getPostId(),
+          img: posts.getImgUrl(),
+          caption: posts.getCaption(),
+          shortcode: posts.getShortCode()
+        }))
+        .filter(post => !!post.img);
+      // this.props.history.push({
+      //   pathname: "/results",
+      //   state: { results: postdata }
+      // });
+      console.log(postdata);
+      this.setState({
+        posts: postdata
+      });
+    });
+  }
+
+  render() {
+    const { user, posts } = this.state;
+    if (!user) {
+      return "no user found";
+    }
+
+    return (
+      <div className="body">
+        <div className="container-card">
+          <div className="column-one-third"></div>
+          <div className="column-one-third">
+            <div className="box">
+              <div>
+                <img className="avatar-image" src={user.avatarurl} />
+              </div>
+              <div className="headline">Hello {user.realname}</div>
+              <div className="sub-headline" key={user.username}>
+                <a
+                  target="_blank"
+                  href={"https://instagram.com/" + user.username}
+                >
+                  @{user.username}
+                </a>
+              </div>
+              <div className="body-text">{user.bio}</div>
+            </div>
+          </div>
+          <div className="column-one-third"></div>
+        </div>
+
+        <div className="container-plattforms">
+          <div className="column-one-sixed"></div>
+          <div className="column-four-sixed">
+            <div className="social-icon-box">
+              <div className="sub-headline">
+                Your public Social Media Accounts:
+              </div>
+              <div>
+                <a
+                  target="_blank"
+                  href={"https://instagram.com/" + user.username}
+                >
+                  <img className="social-icon" src={IGlogo} />
+                </a>
+              </div>
+              <div>
+                <img className="social-icon" src={FBlogo} />
+              </div>
+              <div>
+                <img className="social-icon" src={Twitterlogo} />
+              </div>
+              <div>
+                <img className="social-icon" src={LIlogo} />
+              </div>
+            </div>
+          </div>
+          <div className="column-one-sixed"></div>
+        </div>
+
+        <div className="container-plattforms">
+          <div classNme="column-one-sixed"></div>
+          <div className="column-four-sixed-posts">
+            {posts.map(post => (
+              <IGPost key={post.shortcode} post={post} />
+            ))}
+          </div>
+          <div classNme="column-one-sixed"></div>
+        </div>
+      </div>
```

Comment:
That are a lot of `div`-elements. Is there some way to avoid that? Or to format it more nicely?

Reply:
we could use grid components or bootstrap implement rows and columns as own components.